### PR TITLE
Reduced usage of batch in python samples

### PR DESCRIPTION
**Pull request by mryzhov**
And corrected input arguments

### CODE REVIEW

**Review by generalova-kate — APPROVED** [review]

File: inference-engine/ie_bridges/python/sample/hello_classification/hello_classification.py [hunk 1]
```diff
@@ -30,9 +30,8 @@ def build_argparser():
     args.add_argument('-h', '--help', action='help', default=SUPPRESS, help='Show this help message and exit.')
     args.add_argument("-m", "--model", help="Required. Path to an .xml or .onnx file with a trained model.", required=True,
                       type=str)
-    args.add_argument("-i", "--input", help="Required. Path to a folder with images or path to an image files",
-                      required=True,
-                      type=str, nargs="+")
+    args.add_argument("-i", "--input", help="Required. Path to image file.",
```
Comment (akuporos) [inline inference-engine/ie_bridges/python/sample/hello_classification/hello_classification.py:33]: I suppose it should be: Path to ```an``` image file

**Review by akuporos — APPROVED** [review]
Generally looks good

File: inference-engine/ie_bridges/python/sample/hello_classification/hello_classification.py (hunk 1 above)
Comment (mryzhov) [reply inference-engine/ie_bridges/python/sample/hello_classification/hello_classification.py:33]: yeh, good catch ) will fix

Comment (mryzhov) [reply inference-engine/ie_bridges/python/sample/hello_classification/hello_classification.py]: fixed

**Pull request merged**